### DISCUSSION TITLE
Particle sintering problem

### DISCUSSION THREAD

**Discussion by HPL001** [idaholab/moose#29372] [Q&A Modules: Phase field]
Hello, dear researchers. I want to use moose to simulate the sintering process of two particles, mainly considering the phase field model, elasticity, etc. May I ask if there are any tutorials or existing cases that I can learn

**Comment by GiudGiud** [comment]
Hello

There are a few examples of sintering models in the Phase field module
https://mooseframework.inl.gov/source/materials/ElectrochemicalSinteringMaterial.html
see https://mooseframework.inl.gov/source/materials/GrandPotentialSinteringMaterial.html
which has a list of examples at the bottom of the page

more info on the grand potential model here
https://mooseframework.inl.gov/source/actions/GrandPotentialKernelAction.html

**Reply by HPL001** [reply]
Thank you for your help. In fact, what I want to understand is the simple sintering of two particles. I have seen some papers using MOOSE（for example，https://link.springer.com/article/10.1007/s10853-017-1846-3）, but I do not know how they do it. Can you give me some advice?

**Reply by GiudGiud** [reply]
This might be in Crow since it's @SudiptaBiswas 's research
